src/main/java/software/amazon/smithy/lsp/SmithyLanguageServer.java

```diff
@@ -64,10 +64,11 @@ private void loadSmithyBuild(File root) throws ValidationException, FileNotFound
       File smithyBuild = Paths.get(root.getAbsolutePath(), file).toFile();
       if (smithyBuild.isFile()) {
         try {
-          result.merge(SmithyBuildLoader.load(smithyBuild.toPath()));
-          LspLog.println("Loaded build extensions " + result + " from " + smithyBuild.getAbsolutePath());
+          SmithyBuildExtensions local = SmithyBuildLoader.load(smithyBuild.toPath());
+          result.merge(local);
+          LspLog.println("Loaded build extensions " + local + " from " + smithyBuild.getAbsolutePath());
```

Comment:
The logging previously was logging the builder, which doesn't have a useful toString representation.